spec/requests/v1/songs_request_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+describe 'songs endpoints' do
+  describe 'GET /index' do
+    it 'returns JSON for all songs' do
```

Comment:
Yes, exactly.

Particularly when you get to multiple edge case tests, this is very useful.  For example, if tests out an endpoint to add a song, might have context blocks like:

```ruby
context 'authenticated user' do
  context 'with valid params' do
    # ...
  end

  context 'missing required attributes' do
    # ...
  end

  context 'when song already exists' do
    # ...
  end
end

context 'unauthenticated user' do
  # ...
end
```